compile-build

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 
-set -e
+set -e # Exit immediately if something fails
+export NODE_ENV=production
+export RAILS_ENV=development
```

Comment:
Why is there this difference between NODE_ENV and RAILS_ENV? This is going to be used to compile a production release

Reply:
I have copied these from Sequencescape.

Reply:
So it might not be needed ... The RAILS_ENV is not the same as deploying. This is generating the assets not the environments.